java/java.editor.base/test/unit/src/org/netbeans/modules/java/editor/base/semantic/DetectorTest.java

```diff
@@ -508,6 +508,62 @@ public void testIncDecReading230408() throws Exception {
         performTest("IncDecReading230408");
     }
 
+    public void testRecord1() throws Exception {
+        //TODO: check records are available, skip if they are not.
```

Comment:
There should be a SourceVersion check, or something along those lines, and the test case should be skipped if the current javac does not support records. Otherwise, the test will fail when running e.g. with nb-javac without records. (Dtto the other occurrence below.)

Reply:
Comment addressed.